packages/editor/src/client/routes/theme.less

```diff
@@ -0,0 +1,3 @@
+.duplicated-row {
+    color: #ff4545 ;
```

Comment:
I suggest you use `background` and maybe also use something like this `transition: background-color 0.3s linear;` to make it more smooth. I copied this from the `:hover` css.
![image](https://user-images.githubusercontent.com/2340416/111126294-d4546a00-8572-11eb-87c2-1166e4e3ddbf.png)